CHANGELOG.md

```diff
@@ -19,3 +19,4 @@ All notable, unreleased changes to this project will be documented in this file.
 
 - Added support for numeric and lower-case boolean environment variables - #16313 by @NyanKiyoshi
 - Fixed a potential crash when Checkout metadata is accessed with high concurrency - #16411 by @patrys
+- Disallow changing voucher usage limit setting when voucher has been already used - #16459 by @zedzior
```

Comment:
Shouldn't be added here. We are adding to `changelog` only PRs that are only added to main